test/IECore/MurmurHashTest.py

```diff
@@ -54,6 +64,17 @@ def testCopyConstructor( self ) :
 		self.assertEqual( h, IECore.MurmurHash( h ) )
 		self.assertNotEqual( h, IECore.MurmurHash() )
 
+	def testFromString( self ) :
+
+		h = IECore.MurmurHash()
+		h.append( 1 )
+		h.append( "hello" )
+
+		s = h.toString()
+		hs = IECore.MurmurHash()
+		hs.fromString( s )
```

Comment:
I think `static MurmurHash fromString();` might provide slightly better ergonomics, so this could just be a one-line `hs = MurmurHash.fromString( s )`.

Reply:
A static `fromString` function would be fine

Reply:
done